docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -84,31 +84,6 @@ If you're upgrading from a previous version with Universal Profiling enabled, se
 IMPORTANT: When upgrading, you must remove all existing profiling data.
 We still recommend upgrading as the latest version contains several improvements and new features.
 
-[discrete]
-[[profiling-upgrade-known-issues]]
-==== Known issue when configuring data ingestion
-
-If your {ecloud} deployment originated from version 7.x or earlier and is currently running a version between 8.9.0 and 8.10.2, you may encounter a problem while trying to enable Universal Profiling.
-
-Specifically, clicking the *Set up Universal Profiling* button triggers an error message that reads `Error while setting up Universal Profiling`.
-Upgrade to version *8.10.3* or more recent to fix this issue.
-
-
-If you previously enabled Universal Profiling in {ecloud} version 8.9.0, and you're re-executing the setup, you may also encounter

```

Comment:
In theory, this still happen when upgrading from an older version to 8.15 ("If upgrading does not solve the issue, complete the following step").
I am thinking of https://github.com/elastic/sdh-elasticsearch/issues/8206, especially
```
[instance-0000000003] Aborting index creation as index [.profiling-returnpads-private-v001] is considered too old.
```